operate/services/service.py

```diff
@@ -377,10 +376,11 @@ def load(cls, path: Path) -> "Deployment":
         """Load a service"""
         return super().load(path)  # type: ignore
 
+    # pylint: disable=too-many-locals
     def _build_docker(
         self,
         force: bool = True,
-        chain_id: str = "100",
+        chain_id: str = "100",  # pylint: disable=unused-argument
```

Comment:
Why disable and not remove instead?

Reply:
removed in next push

Reply:
Hmm I still cannot see where the `_chain_id` is used in the method.